dissect/target/plugins/os/unix/linux/proc.py

```diff
@@ -140,16 +137,19 @@ class PacketSocket:
     owner: str = field(default=None)  # resolved owner from user (uid) field
```

Comment:
If you're only specifying the default value, would this not work as well (and be a little cleaner)?

```suggestion
    owner: Optional[str] = None # resolved owner from user (uid) field
```